generator/search_index_builder.cpp

```diff
@@ -100,6 +100,8 @@ void GetCategoryTypes(CategoriesHolder const & categories, pair<int, int> const
                       feature::TypesHolder const & types, vector<uint32_t> & result)
 {
   Classificator const & c = classif();
+  static const uint32_t wheelchair = c.GetTypeByPath({ "wheelchair" });
+  static const uint32_t internet = c.GetTypeByPath({ "internet_access" });
```

Comment:
Please, implement a named class in the same way as other other subclasses of BaseChecker, say:
```c++
class SpecialSearchTypes : public ftypes::BaseChecker
{
  ...
};
```
and use it here.